templates/logs.html

```diff
@@ -0,0 +1,26 @@
+<!doctype html>
+<html>
+    <head>
+        <link href="../../bootstrap.min.css" rel="stylesheet">
+    </head>
+
+    <body>
+        <div class="container-fluid">
+            <div class="col-md-10 col-md-offset-1">
```

Comment:
Columns must be wrapped in a [`.row`](http://getbootstrap.com/css/#grid-nesting).